pages/Documents/Reporting/AgentActivityAPI/overview.md

```diff
@@ -13,6 +13,12 @@ This API is currently in an early adoption phase. To participate in this program
 {: .notice}
 This API is intended for reporting and information purposes, not for real-time decisions, such as routing.
 
+
+{: .notice}
```

Comment:
If breaks are really needed, use HTML for the entire notification:

```
<div class="notice">
  <p>…</p>
  <p>…</p>
  <p>…</p>
</div>
```

Otherwise, remove the breaks and work with one paragraph.

Reply:
@j9t Done